docs/sdk/api/typescriptSdk.md

```diff
@@ -25,6 +25,126 @@ npm install --save @commercetools/ml-sdk
+
+### Getting a client credentials to use the commercetools API
+From the API client Details page mentioned in the previous step copy  `project_key`, `clientId`, `clientSecret`, `API URL`, `scope` and `Auth URL`. The commercetools API uses `clientId` and `clientSecret` to authenticate the requests your app makes. In a later step, you’ll be asked to use these values in your code.
+
+### Set up your local project
+If you don’t already have a project, let’s create a new one. In an empty directory, you can initialize a new project using the following command:
+
+```
+$ npm init -y
+```
+
+After you’re done, you’ll have a new package.json file in your directory.
+Install the `@commercetools/sdk-client`, `@commercetools/sdk-middleware-auth`, `@commercetools/sdk-middleware-http`, `@commercetools/typescript-sdk` and `dotenv` packages and save it to your `package.json` dependencies using the following command:
+
+```
+$ npm install @commercetools/sdk-client @commercetools/sdk-middleware-auth @commercetools/sdk-middleware-http @commercetools/typescript-sdk dotenv
+```
+
+Create a new file called `project.js` in this directory and add the following code:
+```js
+const { createClient } = require('@commercetools/sdk-client')
+const { createAuthMiddlewareForClientCredentialsFlow } = require('@commercetools/sdk-middleware-auth')
+const { createHttpMiddleware } = require('@commercetools/sdk-middleware-http')
+const { createApiBuilderFromCtpClient } = require("@commercetools/typescript-sdk");
+  
+const fetch = require('node-fetch')
+require('dotenv').config()
+
+console.log('Getting started with commercetools Typescript SDK');
+```
+
+Back at the command line, run the program using the following command:
+```
+$ node project.js
+Getting started with commercetools Typescript SDK
+```
+If you see the same output as above, we’re ready to start.
+
+### Get commercetools project settings with the commercetools API
+In this guide we’ll get project settings information. We’ll also follow the best practice of keeping secrets outside of your code (do not hardcode sensitive data).
+
+Store the **client id** and **secret** in a new environment variable. Create a new file called `.env` in this directory and add the following code: 
+
+```
+ADMIN_CLIENT_ID=<your_admin_client_id>
+ADMIN_CLIENT_SECRET=<your_admin_secret_id>
+```
+Replace the values with your client id and client secret that you copied earlier.
+
+Re-open `project.js` and add the following code:
+```js
+const { 
+    ADMIN_CLIENT_ID,
+    ADMIN_CLIENT_SECRET,
+} = process.env;
+
+const projectKey = '<your_project_key>'
+
+// Create a httpMiddleware for the your project AUTH URL
+const authMiddleware = createAuthMiddlewareForClientCredentialsFlow({
+    host: '<your_auth_url>',
+    projectKey,
+    credentials: {
+        clientId: ADMIN_CLIENT_ID,
+        clientSecret: ADMIN_CLIENT_SECRET,
+    },
+    scopes: ['<your_client_scope>'],
```

Comment:
Could we read this from the `.env` file as well? Then this part does not have to be edited by the reader.